src/public/modules/forms/services/form-logic.client.service.ts

```diff
@@ -0,0 +1,53 @@
+import {
```

Comment:
Should this whole file be in a `form-logic.constants` file instead? There are no functions in this file at all.

Reply:
sure, can I do that later? I intend to refactor this further by determining which functions from the controllers should actually belong in the services, but didn't have time over the weekend..

Reply:
i've expanded the file to include functions, it's definitely a service now